BuildAssets/test-linux-packages.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# This script requires environment variables to run:
+#   OCTOPUS_CLI_SERVER, OCTOPUS_CLI_API_KEY, OCTOPUS_SPACE, OCTOPUS_EXPECT_ENV
+# You must also supply either:
+#   TEST_QUICK (to skip some distributions)
+# or
+#   REDHAT_SUBSCRIPTION_USERNAME, REDHAT_SUBSCRIPTION_PASSWORD (to support RHEL tests)
+
+TEST_DEB_SH='
+  # Configure apt
+  export DEBIAN_FRONTEND=noninteractive
```

Comment:
I've read this is discouraged... https://docs.docker.com/engine/faq/#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles
Is that relevant to us?

Reply:
Interesting. I think it will be okay when we're not building an image or will ever run interactively.